### PR TITLE
Automated cherry pick of #2446: Use the right registry for OSS cni-fips

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2446 on release-v1.28.

#2446: Use the right registry for OSS cni-fips

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
